bin/setup.sh

```diff
@@ -71,3 +71,30 @@ else
   rm chromedriver_linux64.zip
   cd $APP_PATH
 fi
+
+# Install Chrome
+if type google-chrome > /dev/null; then
+  echo "google-chrome already installed, skipping install"
+else
+  echo "google-chrome does not exit on this system, installing"
+  mkdir -p google-chrome
+
+  if [[ -f google-chrome/google-chrome-stable_current_amd64.deb ]]; then
+    echo "google-chrome .deb file already exists, skipping download"
+  else
+    echo "Getting google-chrome from repository..."
+    cd google-chrome
+    # wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add - # Get apt installation key
+    wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```

Comment:
**Note:** Installation of chrome is expected to be for a Debian based Linux distro. So it will work for Debian and Ubuntu, but won't work for CentOS, Redhat, and others.